### PR TITLE
removed openbci from vendor.js

### PR DESCRIPTION
**Pull request by tashoecraft**
one line fix to get rid of this bug.

### CODE REVIEW

File: client/vendor.js [hunk 1]
```diff
@@ -11,5 +11,4 @@ module.exports = function() {
     require('script!jspsych/jspsych.js');
     require('script!jspsych/plugins/jspsych-text.js');
     require('script!jspsych/plugins/jspsych-single-stim.js');
-    require('script!openbci-sdk');
```
Comment (andrewjaykeller) [inline client/vendor.js:14]: Where is this now being required?

Comment (andrewheusser) [inline client/vendor.js:14]: its being required in the factory

**Comment by andyh616** [comment]
@teon, can you give me write access to I can merge this?  Thanks!

**Comment by andrewheusser** [comment]
@teonlamont can you give me write access so I can merge this?  Thanks!

**Comment by teonbrooks** [comment]
Sorry @andyh616. When I moved this to a standalone project, I somehow missed you :sadface: you're good to go now!!

**Pull request merged**